docs/source/quickstart.rst

```diff
@@ -5,7 +5,7 @@ Quick Start
 Installation
 --------------
 
-- For x86 Linux or Mac, 
+- For x86 Linux, 
```

Comment:
I suggest to keep intel mac here, so that the user dont need to refer to further docs

Reply:
This is not specifically the gpu version of tensorflow on MacOS. That version is tensorflow-metal and not installed in this tutorial and it is indeed slower than apple optimized version of tensorflow.
A benchmark is underway.

Reply:
A benchmark result have been attached to the end of the latest version of the install guide